### PR TITLE
set timeout of raft api test to 60s

### PR DESCRIPTION
**Pull request by javeme**
Change-Id: Ifa93e46992e79cc9b0adc93ad529b72169ec7a7f

### CODE REVIEW

**Comment by javeme** [comment]
raft ci error:
```java
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.baidu.hugegraph.api.ApiTestSuite
Error:  Tests run: 53, Failures: 4, Errors: 33, Skipped: 0, Time elapsed: 74.035 s <<< FAILURE! - in com.baidu.hugegraph.api.ApiTestSuite
Error:  testEliminate(com.baidu.hugegraph.api.IndexLabelApiTest)  Time elapsed: 0.114 s  <<< FAILURE!
java.lang.AssertionError: Response with status 400 and content {"exception":"class com.baidu.hugegraph.exception.ExistedException","message":"The index label 'personByAge' has existed","cause":""} expected:<202> but was:<400>
	at com.baidu.hugegraph.api.IndexLabelApiTest.testEliminate(IndexLabelApiTest.java:94)

Error:  testEliminate(com.baidu.hugegraph.api.IndexLabelApiTest)  Time elapsed: 0.115 s  <<< ERROR!
com.baidu.hugegraph.HugeException: Failed to list indexlabels

Error:  testAppend(com.baidu.hugegraph.api.IndexLabelApiTest)  Time elapsed: 0.038 s  <<< FAILURE!
java.lang.AssertionError: Response with status 400 and content {"exception":"class com.baidu.hugegraph.exception.ExistedException","message":"The index label 'personByAge' has existed","cause":""} expected:<202> but was:<400>
	at com.baidu.hugegraph.api.IndexLabelApiTest.testAppend(IndexLabelApiTest.java:66)
```

**Comment by javeme** [comment]
unit test error"
```java
Error:  Tests run: 391, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 628.336 s <<< FAILURE! - in com.baidu.hugegraph.unit.UnitTestSuite
Error:  testMultiThreadsGetAndUpdateWithGtCapacity(com.baidu.hugegraph.unit.cache.CacheTest$RamCacheTest)  Time elapsed: 27.98 s  <<< ERROR!
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.AssertionError
Caused by: java.util.concurrent.ExecutionException: java.lang.AssertionError
Caused by: java.lang.AssertionError
```

**Comment by javeme** [comment]
hbase test error:
```java
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.baidu.hugegraph.api.ApiTestSuite
Error:  Tests run: 62, Failures: 3, Errors: 28, Skipped: 0, Time elapsed: 14.642 s <<< FAILURE! - in com.baidu.hugegraph.api.ApiTestSuite
Error:  testList(com.baidu.hugegraph.api.TaskApiTest)  Time elapsed: 0.111 s  <<< FAILURE!
java.lang.AssertionError: Response with status 400 and content {"exception":"class com.baidu.hugegraph.exception.ExistedException","message":"The index label 'personByCity' has existed","cause":""} expected:<202> but was:<400>
	at com.baidu.hugegraph.api.TaskApiTest.prepareSchema(TaskApiTest.java:42)

Error:  testList(com.baidu.hugegraph.api.TaskApiTest)  Time elapsed: 0.111 s  <<< ERROR!
com.baidu.hugegraph.HugeException: Failed to list indexlabels

Error:  testCancel(com.baidu.hugegraph.api.TaskApiTest)  Time elapsed: 0.03 s  <<< FAILURE!
java.lang.AssertionError: Response with status 400 and content {"exception":"class com.baidu.hugegraph.exception.ExistedException","message":"The index label 'personByCity' has existed","cause":""} expected:<202> but was:<400>
	at com.baidu.hugegraph.api.TaskApiTest.prepareSchema(TaskApiTest.java:42)

Error:  testCancel(com.baidu.hugegraph.api.TaskApiTest)  Time elapsed: 0.033 s  <<< ERROR!
com.baidu.hugegraph.HugeException: Failed to list indexlabels

Error:  testDelete(com.baidu.hugegraph.api.TaskApiTest)  Time elapsed: 0.029 s  <<< FAILURE!
java.lang.AssertionError: Response with status 400 and content {"exception":"class com.baidu.hugegraph.exception.ExistedException","message":"The index label 'personByCity' has existed","cause":""} expected:<202> but was:<400>
	at com.baidu.hugegraph.api.TaskApiTest.prepareSchema(TaskApiTest.java:42)

Error:  testDelete(com.baidu.hugegraph.api.TaskApiTest)  Time elapsed: 0.029 s  <<< ERROR!
com.baidu.hugegraph.HugeException: Failed to list indexlabels

Error:  com.baidu.hugegraph.api.GremlinApiTest  Time elapsed: 0.082 s  <<< ERROR!
com.baidu.hugegraph.HugeException: Failed to list indexlabels
```

File: hugegraph-test/src/main/java/com/baidu/hugegraph/api/TaskApiTest.java [hunk 1]
```diff
@@ -52,10 +52,17 @@ public void testList() {
...
+         *        status is SUCCESS, which is stored in DB if there are worker
+         *        nodes in raft-api test.
+         * NOTE: seems the master node won't store task status in memory,
+         *       because only worker nodes store task status in memory.
+         */
         r = client().get(path, ImmutableMap.of("status", "RUNNING"));
         content = assertResponseStatus(200, r);
         tasks = assertJsonContains(content, "tasks");
-        Assert.assertTrue(tasks.isEmpty());
+        Assert.assertTrue(tasks.toString(), tasks.isEmpty());
```
Comment (javeme) [inline hugegraph-test/src/main/java/com/baidu/hugegraph/api/TaskApiTest.java:65]: may be caused by:
- raft async apply?
- update status after task.done()?

**Review by imbajin — APPROVED** [review]

**Review by coderzc — APPROVED** [review]

**Pull request merged**